tools/generators/recipe-generator/files/__packageName__/__fileName__.stories.mdx__tmpl__

```diff
@@ -0,0 +1,52 @@
+import LinkTo from '@storybook/addon-links/react';
+import { Example } from '../../templates';
```

Comment:
Is this supposed to import `TemplateExample` instead of `Example`?

Reply:
ah you're right